malboxes/malboxes.py

```diff
@@ -565,6 +572,42 @@ def document(profile_name, modtype, docpath, fd):
 
     fd.write(line)
 
+def shortcut_function(fd):
+    """ Add shortcut function to the profile """
+    fd.write("""function Add-Shortcut{
```

Comment:
I think we should extract that code in an external `.ps1` file that we can inline at runtime. Add it in here: `malboxes/scripts/windows/`

Then you can replace shortcut_function(fd) with an open(...).read()